lntest/itest/lnd_test.go

```diff
@@ -10213,14 +10209,18 @@ func testMultiHopHtlcLocalTimeout(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("bob should have pending htlc but doesn't")
 	}
 
-	// Now we'll mine an additional block, which should include the second
-	// layer sweep tx.
-	block := mineBlocks(t, net, 1, 1)[0]
+	// Now we'll mine an additional block, which should confirm Bob's commit
+	// sweep. This block should also prompt Bob to broadcast their second
```

Comment:
Why is Bob broadcasting the second layer sweep here? Is the second-layer sweep also waiting for exactly `defaultCSV`? Add a comment.

Reply:
👍

Reply:
@halseth I think this has to do with the outgoingbroadcastdelta being 0.